src/client/utils/Helpers.js

```diff
@@ -1,12 +1,20 @@
+export const prettifyDate = (date) => {
```

Comment:
FYI, this prettifyDate is going away with the latest change Im making. We're replacing them with Moment.js functions.